go/test/endtoend/backup/vtbackup/backup_only_test.go

```diff
@@ -367,3 +374,39 @@ func verifyDisableEnableRedoLogs(ctx context.Context, t *testing.T, mysqlSocket
 		}
 	}
 }
+
+// This helper function wait for all replicas to catch-up the replication.
+// It does this by querying the status detail url of each replica and find the lag.
+func waitForReplicationToCatchup(tablets []cluster.Vttablet) bool {
+	endTime := time.Now().Add(time.Second * 30)
+	timeout := time.After(time.Until(endTime))
+	// key-value structure returned by status url.
+	type kv struct {
+		Key   string
+		Class string
+		Value string
+	}
+	// defining a struct instance
+	var statuslst []kv
+	for {
+		select {
+		case <-timeout:
+			return false
```

Comment:
We should either require.Fatal here or use require.True() with the return value in the calling location (it's currently ignored by the caller).